packages/rnmodules/card/CardManager.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+type CardType = {
+  cardholder: string,
+  expiryMonth: string,
+  expiryYear: string,
+  number: string,
+};
+
+export const getCard = (): Promise<CardType> | null => {
+  return NativeModules.RNCardManager.getCard();
+};
+
+export const saveCard = (card: CardType) => {
+  NativeModules.RNCardManager.saveCard(card);
```

Comment:
Could this return some kind of indication on whether the save was successful or not?